src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
+
+import core.basesyntax.storage.Storage;
+import org.junit.After;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private static FruitDao fruitDao = new FruitDaoImpl();
+
+    @After
+    public void tearDown() {
+        Storage.fruits.clear();
+    }
+
+    @Test
+    public void add_NullFruitName_NotOk() {
+        fruitDao.add(null, 10);

```

Comment:
usually for notOk tests we expect some exception. And I think we should get some exception in case we trying to add null fruit(samo for other NotOk tests in this class)